### DISCUSSION TITLE
Storage and visualisation of data that not related to test results

### DISCUSSION THREAD

**Discussion by ZAKOVYIKA** [allure-framework/allure2#2302] [TestOps]
Greetings!

I wonder if Allure TestOps has the ability to:
1) save data that does not relate to tests, represented, for example, by a csv-file.
2) use saved data to build graphs in dashboards.

To better understand the idea, I will give the following example. 
There is data on the hours of work of employees in the sprint under the contract, as well as information on the amount of story points that employees completed during the sprint. It would be interesting to store this data and display it as a bar graph, for example, to track productivity changes from sprint to sprint (pic.1.).

I will be glad to see any examples, ideas and links to documentation.

![image](https://github.com/allure-framework/allure2/assets/127024282/9cf75129-3177-45a8-8d85-bea347906677)
Pic.1.: Example of desired result.

**Comment by karlisV** [comment]
I know it's pretty late but posting in case someone else stumbles upon this requirement. I have created an Allure report plugin for exactly this purpose, feel free to check it out here https://github.com/karlisV/allure-graphs-plugin
